rustls/src/error.rs

```diff
@@ -218,6 +218,9 @@ pub enum Error {
     /// The peer didn't give us any certificates.
     NoCertificatesPresented,
 
+    /// The presented server name wasn't valid for certificate validation.
```

Comment:
I suggest "The certificate verifier doesn't support the given type of name."

Reply:
👍 Thank you.